downstream/modules/hub/proc-edit-namespace.adoc

```diff
@@ -4,18 +4,18 @@
 
 = Adding additional information and resources to a namespace
 
-You can add information and provide resources for your users to accompany collections included in the namespace. Add a logo and a description, and link users to your GitHub repository, issue tracker, or other online assets. You can also enter markdown text in the *Edit resources* tab to include more information. This is helpful to users who use your collection in their automation tasks.
+You can add information and provide resources for your users to accompany collections included in the namespace. For example, you can add a logo and a description, and link users to your GitHub repository, issue tracker, or other online assets. You can also enter markdown text in the *Edit resources* tab to include more information. This is helpful to users who use your collection in their automation tasks.
 
 .Prerequisites
 
 * You have *Change Namespaces* permissions.
 
 .Procedure
 //[ddacosta] For 2.5 this will be Log in to Ansible Automation Platform and select Automation Content. Automation hub opens in a new tab. From the navigation ...
-. Log in to your {PrivateHubName}.
+. Log in to {PlatformNameShort}.
 . From the navigation panel, select {MenuACNamespaces}.
 . Click the btn:[More Actions] icon *{MoreActionsIcon}* and select *Edit namespace*.
-. In the *Edit details* tab, enter information in the fields.
+. In the *Edit details* tab, enter the relevant information in the fields.
 . Click the *Edit resources* tab to enter markdown in the text field.
```

Comment:
This is a field not a tab. Suggest changing to:
Enter markdown in the *Resources* field.